src/main/java/com/offdk/play/model/SlackCommand.java

```diff
@@ -5,9 +5,12 @@
   private String teamId;
   private String teamDomain;
   private String channelId;
+  private String channelName;
   private User commandUser;
+  private String command;
   private String text;
   private String responseUrl;
+
   public String getToken() {
```

Comment:
We should probably discuss using a style guide to standardize formatting.

[Use google's?](https://github.com/google/styleguide)